dashboard/app/views/layouts/_header.html.haml

```diff
@@ -98,7 +98,7 @@
                 = render file: Rails.root.join('..', 'shared', 'haml', 'hamburger.haml'), locals: hamburger_options
 
 - if show_header2
-  - if request.path == "/" || request.path == "/courses"
+  - if request.path == "/home" || request.path == "/courses"
```

Comment:
I see this isn't new in this PR, so no need to change it, but why do we have this logic that inspects the request.path in a partial view? It would be cleaner to have the parent view pass in a param for the header-banner image.